contrib/lib/src/space_helmet/helmet.rs

```diff
@@ -0,0 +1,287 @@
+use std::sync::atomic::{AtomicBool, Ordering};
+
+use rocket::fairing::{Fairing, Info, Kind};
+use rocket::http::Header;
+use rocket::{Request, Response, Rocket};
+
+use super::policy::*;
+
+/// SpaceHelmet implements a [Fairing](../../rocket/fairing/trait.Fairing.html)
```

Comment:
Type names should be in ticks: `SpaceHelmet`, [`Fairing`], and so on. Also, links to other documentation should be absolute, otherwise we can't move this module around easily: `/rocket/fairing/trait.Fairing.html`. Better yet, you should try using automatic inter-doc linking. It's possible that just [`Fairing`] works.